### PR TITLE
Use NDSForwarder for creating forwarders from the SD card

### PR DESCRIPTION
**Pull request by hansoljin**
The forwarder guide now uses https://github.com/MechanicalDragon0687/NDSForwarder for forwarding from the SD card. 

To work this out, I used the pre-existing SD card and Flashcard tabs, moved all existing content to the "Flashcard" tab (as that hasn't changed), and wrote up a new guide for the SD card tab.

Please provide feedback.

### CODE REVIEW

File: pages/_en-US/ds-index/3ds-forwarders.md [hunk 1]
```diff
@@ -13,12 +13,39 @@ tabs:
...
+This page assumes you are running a modern CFW environment from [3ds.hacks.guide](https://3ds.hacks.guide).
+
+Choose one of the following to add to the HOME menu:
+
+{% capture tab-sd-card %}
+
+### Part 1: Getting the required files
+1. Open FBI and select `Remote Install`, then `Scan QR Code`
+1. Scan this QR code to install the latest version of [Universal-Updater](https://github.com/Universal-Team/Universal-Updater)<br>
+   ![Universal-Updater QR code](https://db.universal-team.net/assets/images/qr/universal-updater.cia.png)
```
Comment (Epicpkmn11) [inline pages/_en-US/ds-index/3ds-forwarders.md:30]: Might be good to mention that you can skip this if you already have Universal-Updater as a lot of people probably already have it considering it's on 3ds.hacks.guide now and the default way to install TWiLight.

File: pages/_en-US/ds-index/3ds-forwarders.md [hunk 2]
```diff
@@ -80,17 +107,6 @@ PC:
 - **Linux users:** JavaFX. On Debian based systems run [this](https://gist.githubusercontent.com/puntillol59/7532b6583380baca236dcaf2d8f75b5c/raw/e8b9d193f8b24de941160c7292ec0bb3b997e98e/main.sh), or if you're on Arch run: `sudo pacman -S java8-openjfx && sudo archlinux-java set java-8-openjdk/jre`.
```
Comment (Epicpkmn11) [inline pages/_en-US/ds-index/3ds-forwarders.md:107]:
```suggestion
- **Linux users:** JavaFX
   - Debian based: Run [this](https://gist.githubusercontent.com/puntillol59/7532b6583380baca236dcaf2d8f75b5c/raw/e8b9d193f8b24de941160c7292ec0bb3b997e98e/main.sh)
   - Arch: `sudo pacman -S java8-openjfx && sudo archlinux-java set java-8-openjdk/jre`
```
Oh, this wasn't changed here so might just commit this separately but I think this looks better while on the topic of changing this page.

File: pages/_en-US/ds-index/3ds-forwarders.md [hunk 3]
```diff
@@ -13,12 +13,39 @@ tabs:
 If you have any issues, check the FAQs on the [GBAtemp thread](https://gbatemp.net/threads/nds-forwarder-cias-for-your-home-menu.426174/).
```
Comment (Epicpkmn11) [inline pages/_en-US/ds-index/3ds-forwarders.md:13]: Should this be moved to the flashcard tab?

**Comment by hansoljin** [comment]
Amended.

**Pull request merged**